src/main/app/src/app/zaken/zaken-mijn/zaken-mijn.component.html

```diff
@@ -4,205 +4,174 @@
   -->
 
 <div class="table-wrapper" [class.loading]="utilService.loading$ | async">
-    <table mat-table matSort multiTemplateDataRows
-           cdkDropList
-           cdkDropListLockAxis="x"
-           cdkDropListOrientation="horizontal"
-           (cdkDropListDropped)="dataSource.drop($event)">
+    <table mat-table matSort matSortDisableClear [matSortDirection]="dataSource.zoekParameters.sorteerRichting"
+           [matSortActive]="dataSource.zoekParameters.sorteerVeld"
+           multiTemplateDataRows cdkDropList cdkDropListLockAxis="x" cdkDropListOrientation="horizontal" (cdkDropListDropped)="dataSource.drop($event)">
 
         <ng-container matColumnDef="zaak.identificatie_filter">
-            <th mat-header-cell *matHeaderCellDef></th>
+            <th mat-header-cell *matHeaderCellDef>
+                <zac-tekst-filter (changed)="dataSource.zoekParameters.zoeken['IDENTIFICATIE'] = $event; filtersChange()"></zac-tekst-filter>
```

Comment:
Is er niet een manier om het gebruik van al die strings 'IDENTIFICATIE', 'ZAAK_STATUS', 'ZAAK_ZAAKTYPE' enz in een enum te ztten? Of er constanten van te maken?
Alleen als het nakkelijk kan.